app/Http/Controllers/Maps/GroupController.php

```diff
@@ -34,14 +34,34 @@ public function index(Campaign $campaign, Map $map)
             ->groups()
             ->sort(request()->only(['o', 'k']), ['position' => 'asc'])
             ->with(['map'])
-            ->paginate(config('limits.maps_pagination'));
+            ->paginate(config('limits.pagination'));
+
+
+        $groups = $map
+            ->groups()
+            ->sort(request()->only(['o', 'k']), ['position' => 'asc'])
```

Comment:
The user shouldn't be able to change the sort order for the reorder bit, it should be fixed